language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/docs/DocAttachmentInfo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.command.docs;
+
+import org.eclipse.lsp4j.Position;
+
+/**
+ * Holds the meta information required for the documentation attachment.
+ */
```

Comment:
Shall we add the version?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/11445/commits/0952ce385463bb74b166a5494212748227c02b60